### ISSUE TITLE
security holding package

### ISSUE THREAD

**Issue by DanielRuf** [dnnsoftware/Dnn.AdminExperience#1008]
Hi,

it seems you have accidentally added `fs` but this is not a package on npmjs, `fs` is an internal NodeJS module.

https://github.com/dnnsoftware/Dnn.AdminExperience/blob/development/Extensions/Content/Dnn.PersonaBar.Extensions/WebApps/Users.Web/src/_exportables/package.json#L30

See https://nodejs.org/api/fs.html

You might want to remove it using `npm uninstall fs`.

**Comment by valadas** [comment]
It is a devDependency, so not delivered. But it can probably be removed anyway unless the build process needs it for something.

**Comment by DanielRuf** [comment]
This `fs` dependency does nothing, see https://unpkg.com/fs/

**Comment by valadas** [comment]
Yeah, I agree, I am just wondering how it got there, maybe that was a workaround to something or accidental. We just need to remove it and confirm it still builds.
